src/main/frontend/db/model.cljs

```diff
@@ -116,17 +116,26 @@
              db-utils/seq-flatten
              distinct)))
 
-(defn get-alias-page
+(defn get-alias-source-page
   [repo alias]
   (when-let [conn (and repo (conn/get-conn repo))]
-    (some->> (d/q '[:find ?page
-                    :in $ ?alias
-                    :where
-                    [?page :page/alias ?alias]]
-                  conn
-                  alias)
-             db-utils/seq-flatten
-             distinct)))
+    (let [pages (->>
+                 (d/q '[:find (pull ?p [*])
```

Comment:
consider extract query into `db.clj`  ?

Reply:
You mean to separate the queries with data transformation, right?